Code/Tools/SceneAPI/SceneCore/Utilities/SceneGraphSelector.cpp

```diff
@@ -312,15 +312,27 @@ namespace AZ
             void SceneGraphSelector::CorrectRootNode(const Containers::SceneGraph& graph,
                 AZStd::set<AZStd::string>& selected, AZStd::set<AZStd::string>& unselected)
```

Comment:
Could both "AZStd::set<AZStd::string>& selected" and "AZStd::set<AZStd::string>& unselected" be sent in as const?